merlin/merlin_templates.py

```diff
@@ -44,60 +44,30 @@
 from merlin.log_formatter import setup_logging
 
 
-LOG = logging.getLogger("merlin-example")
+LOG = logging.getLogger("merlin-templates")
 DEFAULT_LOG_LEVEL = "INFO"
 
 
-def process_example(args):
-    setup_example(args.example, args.path)
-
-
-def example_list(args):
-    print(banner_small)
-    list_examples()
+def process_templates(args):
+    LOG.warning(
+        "The command `merlin-templates` has been deprecated in favor of `merlin example`."
+    )
 
 
 def setup_argparse():
     parser = argparse.ArgumentParser(
         prog="Merlin Examples",
         description=banner_small,
         formatter_class=argparse.RawDescriptionHelpFormatter,
-        epilog=list_examples()
-        + "\nSee merlin-example <command> --help for more info.\n",
-    )
-    parser.add_argument(
-        "-lvl",
-        "--level",
-        action="store",
-        dest="level",
-        type=str,
-        default=DEFAULT_LOG_LEVEL,
-        help="Set the log level. Options: DEBUG, INFO, WARNING, ERROR. "
-        "[Default: %(default)s]",
-    )
-    parser.add_argument(
-        "example", action="store", type=str, help="The name of the example to setup."
     )
-    parser.add_argument(
-        "-p",
-        "--path",
-        action="store",
-        type=str,
-        default=None,
-        help="Specify a path to write the workflow to. Defaults to current "
-        "working directory",
-    )
-    parser.set_defaults(func=process_example)
-
+    parser.set_defaults(func=process_templates)
     return parser
 
 
 def main():
     parser = setup_argparse()
     args = parser.parse_args()
-
-    setup_logging(logger=LOG, log_level=args.level.upper(), colors=True)
-
+    setup_logging(logger=LOG, log_level="INFO", colors=True)
```

Comment:
why is the log_level being changed?

Reply:
oh i see this is deprecated now

Reply:
then set it to use the DEFAULT_LOG_LEVEL variable and set that to be ERROR

Reply:
Addressed